mini-pools-ios/ViewControllers/PoolViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  PoolViewController.swift
+//  mini-pools-ios
+//
+//  Created by Kaplún, Sebi on 12/20/17.
+//  Copyright © 2017 Kaplún, Sebi. All rights reserved.
+//
+
+import UIKit
+
+class PoolViewController: UIViewController {
+
+    @IBOutlet weak var createdLabel: UILabel!
+    @IBOutlet weak var contributionsTableView: UITableView!
+    
+    var pool : PoolModel?
+    var users : [PersonModel] = []
+    var isSyncingContribution = false
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        contributionsTableView.dataSource = self
+        contributionsTableView.delegate = self
+        contributionsTableView.tableFooterView = UIView()
+
+        if let pool = pool {
+            self.title = pool.name
+            createdLabel.text = "Pool created on \(pool.createdAt.shortDate())"
+            if let creator = pool.creator {
+                createdLabel.text = "\(createdLabel.text!) by \(creator.name)"
+            }
+        }
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        UsersWorker.fetchUsers{ allUsers in
+            self.users = allUsers
+        }
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        let vc = segue.destination as! AddContributionViewController
+        vc.users = users
+        vc.poolId = pool!.id
```

Comment:
better not to force unwrap, the segue should not happen if the pool doesn't exist - no?

Reply:
it will always exists

Reply:
4e800b1